resources/concourse-aws-user/main.tf

```diff
@@ -148,6 +148,40 @@ data "aws_iam_policy_document" "policy" {
       "*",
     ]
   }
+
+  # Roles to Create/Edit/Delete Route53 Zone.
+  statement {
```

Comment:
Can we merge this first new statement with the last one just to keep it tidy?
Can they all be restricted to `"arn:aws:route53:::hostedzone/*"`?